### PR TITLE
[ENG-254]CLI feedback bugfix

### PR DESCRIPTION
**Pull request by dmanjunath**
Lots of bugfixes here
- If on linux and libsecret is not installed, prompt dependencies to be installed
- Silence warnings/errors during key import
- Better checking for region codes
- New admin get command that can run arbitrary get commands across any contract eg `earthfast-cli EarthfastRegistry getUSDC` or `earthfast-cli EarthfastNodes getNodeCount 0x00...`
- Set JSON pretty print limit to null so it doesn't show "... n more items" when printing large arrays
- Ability to filter projects by those with active reservation and optionally show reservations for all projects

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
+    "dev": "export NODE_NO_WARNINGS=1 NODE_OPTIONS=--unhandled-rejections=none && npm run build && ./bin/dev",
     "debug": "DEBUG=1 npm run build && ./bin/dev",
     "build": "rm -rf ./dist && tsc",
     "clean": "rm -rf ./dist",
     "postpack": "rm -f oclif.manifest.json",
     "prepack": "npm run build && oclif manifest"
   },
   "bin": {
-    "earthfast": "./bin/run"
+    "earthfast": "export NODE_NO_WARNINGS=1 NODE_OPTIONS=--unhandled-rejections=none && ./bin/run"
```
Comment (SidSethi) [inline package.json:17]: idk if i'm doing smth wrong but this is not working for me locally, and ai is saying this won't work the bin config can only contain a relative path to executable

File: src/commands/admin/get.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import { BlockchainCommand } from "../../base";
+import { getContract, getProvider, pretty } from "../../helpers";
+
+export default class Get extends BlockchainCommand {
```
Comment (SidSethi) [inline src/commands/admin/get.ts:4]: awesome

**Review by SidSethi — COMMENTED** [review]
one possible issue

File: package.json (hunk 1 above)
Comment (dmanjunath) [reply package.json:17]: i'm able to run this locally 
```
ubuntu@ip-172-31-5-74:~/earthfast-cli$ npx earthfast
Command line utility for EarthFast

VERSION
  earthfast-cli/0.9.0 linux-x64 node-v20.18.1

USAGE
  $ earthfast [COMMAND]

TOPICS
  admin        Perform admin functions such as granting roles.
  bundle       Build EarthFast content bundles.
  key          Manage keys for signing transactions.
  node         Create and manage nodes.
  operator     Create and manage operators.
  project      Create and manage projects.
  reservation  Reserve content nodes for a project.

COMMANDS
  help    Display help for earthfast.
  status  Shows the current epoch status.
```

Comment (SidSethi) [reply package.json:17]: on macos

Comment (dmanjunath) [reply package.json:17]: hmm i can run on macos as well
```
earthfast-cli|dm-keytar-deps-bugfix⚡ ⇒ npx earthfast
Command line utility for EarthFast

VERSION
  earthfast-cli/0.9.0 darwin-arm64 node-v20.10.0

USAGE
  $ earthfast [COMMAND]

TOPICS
  admin        Perform admin functions such as granting roles.
  bundle       Build EarthFast content bundles.
  key          Manage keys for signing transactions.
  node         Create and manage nodes.
  operator     Create and manage operators.
  project      Create and manage projects.
  reservation  Reserve content nodes for a project.

COMMANDS
  help    Display help for earthfast.
  status  Shows the current epoch status.
```

**Review by SidSethi — APPROVED** [review]

**Pull request merged**